tests/python/gaia-ui-tests/gaiatest/apps/system/regions/cards_view.py

```diff
@@ -30,26 +34,25 @@ def _close_button_locator(self, app):
     @property
     def cards(self):
         return [Card(self.marionette, card)
-                for card in self.marionette.find_elements(*self._cards_locator)]
+                for card in self.root_element.find_elements(*self._cards_locator)]
 
     @property
-    def is_cards_view_displayed(self):
-        return self.is_element_displayed(*self._task_manager_locator)
+    def is_displayed(self):
+        return self.is_element_displayed(*self._root_locator)
 
     @property
     def is_cards_view_a11y_hidden(self):
```

Comment:
Yes, but since it's accessibility code, I'd prefer to leave it as is (only fixing accessibility code if it would be obviously broken otherwise).